editor/src/utils/utils.ts

```diff
@@ -626,6 +626,10 @@ function assert(errorMessage: string, predicate: boolean | (() => boolean)): voi
   throw new Error(`Assert failed: ${errorMessage}`)
 }
 
+function assertNever(n: never): never {
+  throw new Error(`Expected \`never\`, got ${JSON.stringify(n)}`)
+}
```

Comment:
This is good, but we've stopped using this specific file as it became a massive grab bag for anything that didn't have a specific place. Instead we have moved to using more specific utils files in `core/shared`